class.jetpack-xmlrpc-server.php

```diff
@@ -96,7 +102,7 @@ function provision_xmlrpc_methods() {
 
 	function remote_authorize( $request ) {
 		$user = get_user_by( 'id', $request['state'] );
-		Tracking::record_user_event( 'jpc_remote_authorize_begin', array(), $user );
+		$this->tracking->record_user_event( 'jpc_remote_authorize_begin', array(), $user );
```

Comment:
Would be nice to have a method for retrieving `$this->tracking` - it would make things easier to test (we can easily mock the method), and if that method is public, it would allow us to call it externally.

Reply:
hmm, not sure if I don't prefer to do `$tracking = new Tracking()` every time

Reply:
I have no reservations, feel free to do it 👍

Reply:
We've decided to keep it as-is for now, and if need be, we can always alter. YAGNI!